src/ring_theory/algebra.lean

```diff
@@ -229,6 +229,19 @@ instance module.endomorphism_algebra (R : Type u) (M : Type v)
   commutes' := by { intros, ext, simp },
   smul_def' := by { intros, ext, simp } }
 
+lemma module.endomorphism_algebra_map_apply (R : Type u) (M : Type v)
+  [comm_ring R] [add_comm_group M] [module R M] (a : R) :
+  (algebra_map R (M →ₗ[R] M)) a = a • linear_map.id := rfl
```

Comment:
I'd use `apply` for what was called `apply2` since that applies the endomorphism to something.
```suggestion
lemma module.algebra_map_End_eq_smul_id (R : Type u) (M : Type v)
  [comm_ring R] [add_comm_group M] [module R M] (a : R) :
  algebra_map R (End R M) a = a • linear_map.id := rfl
```